control-plane/plugin/src/resources/blockdevice.rs

```diff
@@ -0,0 +1,210 @@
+use crate::{
+    operations::GetBlockDevices,
+    resources::{
+        utils::{
+            print_table, CreateRows, GetHeaderRow, OutputFormat, BLOCKDEVICE_HEADERS_ALL,
+            BLOCKDEVICE_HEADERS_USABLE,
+        },
+        NodeId,
+    },
+    rest_wrapper::RestClient,
+};
+use async_trait::async_trait;
+use once_cell::sync::OnceCell;
+use openapi::{clients::tower::Error, models::RestJsonError};
+use prettytable::Row;
+use serde::Serialize;
+
+static USED_DISKS: OnceCell<Vec<String>> = OnceCell::new();
```

Comment:
Why do we need this? Can we not add the list of pools on the node to the new-types which we have below?

Reply:
Yes, we don't need this. Thanks for pointing out. Will refactor it,